topdown/cidr_test.go

```diff
@@ -44,3 +45,75 @@ func TestNetCIDRExpandCancellation(t *testing.T) {
 	}
 
 }
+
+func TestNetCIDRIsValid(t *testing.T) {
```

Comment:
I think these would better live in `test/cases/testdata` -- so other implementors can use the same test cases. Like, IR users, or Wasm SDKs.

Reply:
That's a remnant of earlier times. Don't bother 😅

Reply:
I have migrated the tests to there, please take a look.